code/game/objects/items/robot/items/hypo.dm

```diff
@@ -112,13 +112,13 @@
 	 * In most places we add + 1 because we're secretly keeping [max_volume_per_reagent + 1]
 	 * units, so that when this reagent runs out it's not wholesale removed from the reagents
 	 */
-	var/max_volume_per_reagent = 30
+	var/max_volume_per_reagent = 25
 	/// Cell cost for charging a reagent
-	var/charge_cost = 0.05 * STANDARD_CELL_CHARGE
+	var/charge_cost = 0.75 * STANDARD_CELL_CHARGE
```

Comment:
Pr body says 0.5 j to 0.75j (1.5x)
but this is 0.05j to 0.75j (15x)

Reply:
LGTM